dashboard/app/controllers/sessions_controller.rb

```diff
@@ -21,9 +23,25 @@ def new
     super
   end
 
+  # POST /resource/clever_takeover
```

Comment:
Should this be `POST /users/clever_takeover`?

Reply:
I was following the pattern elsewhere in the file here, you're probably right that these should all be changed. Will hit in cleanup.